packages/cognito-module/src/getters.js

```diff
@@ -10,5 +10,25 @@ module.exports = {
     const isMe = accessToken.payload.username === store.user.username
 
     return hasToken && isActive && isMe
+  },
+  getSession: (store = {}) => {
+    const session = store.session
+    return session
+  },
+  getUserSub: (store = {}) => {
+    const userSub = store.user.attributes.sub
```

Comment:
This should check to be sure that attributes exists before accessing a property on it. This can also just return without a variable assignment.

Reply:
This can be applied to the 3 getters below as well.

Reply:
I've added the requested changes, could you tell me if there is anything else to be corrected?

Reply:
Added some additional changes.

Reply:
The docs have been updated as well.